packages/grid/x-data-grid-pro/src/components/headerFiltering/constants.ts

```diff
@@ -0,0 +1,52 @@
+export const OPERATOR_LABEL_MAPPING: { [key: string]: string } = {
+  '=': 'equals',
+  '!=': 'not equals',
+  '>': 'is greater than',
+  '>=': 'is greater than or equal to',
+  '<': 'is less than',
+  '<=': 'is less than or equal to',
+};
+
+export const OPERATOR_SYMBOL_MAPPING: { [key: string]: string } = {
```

Comment:
In these cases, wouldn't it be better to let tsc infer the typing? The infered type would be more strict than just `{ [key: string]: string }`.

Reply:
Thanks for the suggestion, I tried but strict type here doesn't play well with the [type](https://github.com/mui/mui-x/blob/1f0f06e4697a991388d06d4bdf16aa9da4800780/packages/grid/x-data-grid/src/models/gridFilterItem.ts#L24) of `GridFilterItem.operator` which we have to use as a key for this object. Because that way we may have to typecast `GridFilterItem.operator` to an incorrect type (i.e. the strict type inferred by `tsc`) to make `tsc` happy.

What do you think?